ui/src/app/app.component.ts

```diff
@@ -14,7 +14,6 @@ export class AppComponent {
     private readonly authService: AuthService
   ) {}
 
-
```

Comment:
Revert? I'm generally not a fan of adding whitespace-only changes to code commits.

Reply:
Thanks for catching that – reverted.